tests/XHPHelpersTest.php

```diff
@@ -58,76 +60,74 @@ protected function render(): XHPRoot {
     $x = <test:xhphelpers data-foo="bar" />;
-    $this->assertSame('<div data-foo="bar"></div>', $x->toString());
-    $this->assertNotEmpty($x->getID());
-    $this->assertSame('<div id="'.$x->getID().'"></div>', $x->toString());
+    expect($x->toString())->toBeSame('<div data-foo="bar"></div>');
+    expect($x->getID())->toNotBeEmpty();
+    expect('<div id="'.$x->getID().'"></div>')->toBeSame($x->toString());
   }
 
   public function testTransferAttributesAsyncWithHelpers(): void {
     $x = <test:async:xhphelpers data-foo="bar" />;
-    $this->assertSame('<div data-foo="bar"></div>', $x->toString());
-    $this->assertNotEmpty($x->getID());
-    $this->assertSame('<div id="'.$x->getID().'"></div>', $x->toString());
+    expect($x->toString())->toBeSame('<div data-foo="bar"></div>');
+    expect($x->getID())->toNotBeEmpty();
+    expect('<div id="'.$x->getID().'"></div>')->toBeSame($x->toString());
   }
 
   public function testAddClassWithoutHelpers(): void {
     $x = <test:no-xhphelpers class="foo" />;
     $x->addClass("bar");
     $x->conditionClass(true, "herp");
     $x->conditionClass(false, "derp");
-    $this->assertSame('foo bar herp', $x->:class);
-    $this->assertSame("<div></div>", $x->toString());
+    expect($x->:class)->toBeSame('foo bar herp');
+    expect($x->toString())->toBeSame("<div></div>");
   }
 
   public function testAddClassWithHelpers(): void {
     $x = <test:xhphelpers class="foo" />;
     $x->addClass("bar");
     $x->conditionClass(true, "herp");
     $x->conditionClass(false, "derp");
-    $this->assertSame('foo bar herp', $x->:class);
-    $this->assertSame('<div class="foo bar herp"></div>', $x->toString());
+    expect($x->:class)->toBeSame('foo bar herp');
+    expect($x->toString())->toBeSame('<div class="foo bar herp"></div>');
   }
 
   public function testRootClassPreserved(): void {
     $x = <test:with-class-on-root />;
-    $this->assertSame('<div class="rootClass"></div>', $x->toString());
+    expect($x->toString())->toBeSame('<div class="rootClass"></div>');
   }
 
   public function testTransferedClassesAppended(): void {
     $x = <test:with-class-on-root class="extraClass" />;
-    $this->assertSame(
-      '<div class="rootClass extraClass"></div>',
-      $x->toString(),
-    );
+    expect(      $x->toString(),
+)->toBeSame(
+      '<div class="rootClass extraClass"></div>'    );
   }
 
   public function testRootClassesNotOverridenByEmptyString(): void {
     $x = <test:with-class-on-root class="" />;
-    $this->assertSame('<div class="rootClass"></div>', $x->toString());
+    expect($x->toString())->toBeSame('<div class="rootClass"></div>');
   }
 
   public function testNested(): void {
     $x =
       <test:xhphelpers class="herp">
         <test:xhphelpers class="derp" />
       </test:xhphelpers>;
-    $this->assertSame(
-      '<div class="herp"><div class="derp"></div></div>',
-      $x->toString(),
-    );
+    expect(      $x->toString(),
```

Comment:
Yikes. You might want to run `hackfmt --diff` on this.

Reply:
Oops, I reverted the changes to see what was going on with side effects and forgot to run hackfmt again :(